### PR TITLE
Implement [Alert Table] - Update the Notification Column According to New Design

### PR DESCRIPTION
**Pull request by adi-gini**
Jira link:
https://iguazio.atlassian.net/browse/ML-8524

change the icons:
Screenshot 2024-12-05 at 8 39 01

### CODE REVIEW

File: src/components/Table/table.scss [hunk 1]
```diff
@@ -101,8 +101,24 @@
...
+            fill: $burntSienna;
+          }
+
+          svg {
+            width: 14px;
+            height: 14px;
+            padding: 1px;
+            background-color: white;
+            border-radius: 50%;
+          }
```
Comment (mavdryk) [inline src/components/Table/table.scss]: `-notification` isn't part of the common style for the table. These styles should be in the `alertsTableRow.scss`

File: tests/mockServer/data/alerts.json [hunk 2]
```diff
@@ -8,19 +8,35 @@
       "notifications": [
         {
           "kind": "email",
-          "err": ""
+          "err": "",
```
Comment (mavdryk) [inline tests/mockServer/data/alerts.json:11]: rename `err` to `error`

**Review by mavdryk — CHANGES_REQUESTED** [review]

File: src/components/Table/table.scss (hunk 1 above)
Comment (adi-gini) [reply src/components/Table/table.scss]: Done

File: tests/mockServer/data/alerts.json (hunk 2 above)
Comment (adi-gini) [reply tests/mockServer/data/alerts.json:11]: Done

**Comment by pinis-gini-apps** [comment]
@adi-gini based on backend response, the notification error is "err". 

Screenshot 2024-12-12 at 8 33 47 a m

**Review by mavdryk — APPROVED** [review]

**Pull request closed without merging**